src/lib.rs

```diff
@@ -2,7 +2,7 @@ extern crate pest;
 #[macro_use]
 extern crate pest_derive;
 
-mod arc;
+mod wikilink;
```

Comment:
This needs to be `pub mod wikilink` or `WikilinkHandler` has to be re-exported in order for downstream projects to be able to implement it.

Reply:
You're right, thanks for pointing that out!

I wrote this up quite quickly today, so there's a lot of stuff like this that needs to be fixed

Reply:
No worries, that's what code review is for! 🚀